### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by mabdelmalek22**
Add proxy settings to the doc

### CODE REVIEW

**Comment by mabdelmalek22** [comment]
Add Proxy settings

**Comment by get-itips** [comment]
Looks like this is still work in progress @e0i

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -713,6 +713,14 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -ProxyBypass
+List Proxy Bypass desciption and settings 
+
+### -ProxyPacUrl
+List Proxy PacUrldesciption and settings
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:720]:
```suggestion
Lists the proxy .pac file URLs and settings.
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -713,6 +713,14 @@ Default value: None
...
 Accept wildcard characters: False
 ```
+### -ProxyBypass
+List Proxy Bypass desciption and settings 
+
+### -ProxyPacUrl
+List Proxy PacUrldesciption and settings 
+
+### -ProxyServer
+List Proxy Server desciption and settings
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:723]:
```suggestion
Lists the proxy server descriptions and settings. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mabdelmalek22, kindly commit the suggested changes. Also, include the following information for each of these parameters:

Type: 
Parameter Sets: 
Aliases: 
Required: 
Position: 
Default value: 
Accept pipeline input: 
Accept wildcard characters: 

Please see other parameters for formatting. Many thanks.

**Pull request closed without merging**